{"status": "broken", "description": "Initialzing parameters for testing.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestPerception"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "8042-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608218809835, "uuid": "8f87be9b-3263-462f-b699-532f696d985d", "historyId": "effeafc46fba316bab86f962fc856697", "name": "test_set_up", "start": 1608218809828, "fullName": "tests.complex_test.TestPerception#test_set_up", "statusDetails": {"message": "TypeError: first argument must be an iterable of pandas objects, you passed an object of type \"DataFrame\"", "trace": "self = <tests.complex_test.TestPerception instance at 0x7fc50d7e5560>, randomizer = <function _parameters at 0x7fc50d7eb758>\n\n    def test_set_up(self,randomizer):\n        \"\"\"Initialzing parameters for testing.\n        \"\"\"\n        self.config = Configuration()\n>       a = self.config.config_data_frame('nav_end')\n\ntests/complex_test.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests/file_reader/file_reader.py:110: in config_data_frame\n    df3 = pd.concat(df, df2)\n/usr/local/lib/python2.7/dist-packages/pandas/core/reshape/concat.py:228: in concat\n    copy=copy, sort=sort)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pandas.core.reshape.concat._Concatenator object at 0x7fc522fd79d0>\nobjs =                           Configuration Settings                              ...  Selected world environment                                               room\naxis =     Unnamed: 0              Models  X-pos  Y-pos  Z-pos    Q-1    Q-2    Q-3  ...        23  minicoffeetable731 -2.000 -0.000  0.020  0.000  0.000 -0.000  1.000\njoin = 'outer', join_axes = None, keys = None, levels = None, names = None, ignore_index = False, verify_integrity = False, copy = True, sort = None\n\n    def __init__(self, objs, axis=0, join='outer', join_axes=None,\n                 keys=None, levels=None, names=None,\n                 ignore_index=False, verify_integrity=False, copy=True,\n                 sort=False):\n        if isinstance(objs, (NDFrame, compat.string_types)):\n            raise TypeError('first argument must be an iterable of pandas '\n                            'objects, you passed an object of type '\n>                           '\"{name}\"'.format(name=type(objs).__name__))\nE           TypeError: first argument must be an iterable of pandas objects, you passed an object of type \"DataFrame\"\n\n/usr/local/lib/python2.7/dist-packages/pandas/core/reshape/concat.py:244: TypeError"}, "attachments": [{"source": "26677ab7-e5d3-446f-9950-03143aad5da1-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "f282c127-dc64-49ae-8487-29ecb2b11958-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "147712d4-1e20-4785-85c7-1a44aa17b7b1-attachment.txt", "type": "text/plain", "name": "stderr"}]}